src/OrchardCore/Orchard.Environment.Extensions/Manifests/ManifestInfo.cs

```diff
@@ -1,24 +1,54 @@
 ﻿using Microsoft.Extensions.Configuration;
-
+using System.Collections.Generic;
+using System.Linq;
+using System;
 namespace Orchard.Environment.Extensions.Manifests
 {
     public class ManifestInfo : IManifestInfo
     {
         private readonly IConfigurationRoot _configurationRoot;
         private string _type;
-
+        private Lazy<IEnumerable<string>> _tags;
+        private Lazy<Version> _version;
         public ManifestInfo(
             IConfigurationRoot configurationRoot,
             string type)
         {
             _configurationRoot = configurationRoot;
             _type = type;
+            _tags = new Lazy<IEnumerable<string>>(ParseTags);
+            _version = new Lazy<Version>(ParseVersion);
         }
 
         public bool Exists => true;
         public string Name => _configurationRoot["name"];
         public string Description => _configurationRoot["description"];
         public string Type { get { return _type; } }
+        public string Author => _configurationRoot["author"];
+        public string Website => _configurationRoot["website"];
+        public Version Version => _version.Value;
+        public IEnumerable<string> Tags => _tags.Value;
         public IConfigurationRoot ConfigurationRoot => _configurationRoot;
+
+        private IEnumerable<string> ParseTags()
+        {
+            var tags = _configurationRoot["tags"];
+
+            if (string.IsNullOrWhiteSpace(tags))
+                return Enumerable.Empty<string>();
+
+            return tags.Split(new[] { ',', ' ' }, StringSplitOptions.RemoveEmptyEntries).Select(x => x.Trim()).ToArray();
+        }
+        private Version ParseVersion()
+        {
+            var value = _configurationRoot["version"];
+
+            if (string.IsNullOrWhiteSpace(value))
+                return new Version(0, 0);
+
+            Version version;
+            Version.TryParse(value, out version);
+            return version;
+        }
```

Comment:
Hmmmm hang on... I need to think about this, I have been trying not to introduce these in here.